### ISSUE TITLE
2014er F36 verbindet sich nicht.

### ISSUE THREAD

**Issue by vocaris9** [BimmerGestalt/AAIdrive#637]
Hallo. Ich habe einen F36 420d xdrive Grand Coupé Luxery mit einer NTB Headunit.
Bei mir wird keine BT Verbindung zu meinem Android OnePlus 9 (Android 12) hergestellt.
Ich habe die MyBMW App installiert sowie die AAidrive App. Das Handy hat eine "normal" BT Verbindung zum BMW, um Musik zu hören oder zu telefonieren.
Im GitHub Einleitungstext steht: 
"Jeder BMW oder Mini ab Modelljahr 2014, der mit NBT oder NBT Evo und der Funktion „BMW Apps (6NR)“, einem aktiven [BMW ConnectedDrive-Abonnement oder der Option „Mini Connected (SA6NM)“ ausgestattet ist, sollte kompatibel sein."
Ich habe einen 2014 mit NTB dann folgt aber ein UND.... BMW Apps sehe ich im Bereich Connected Drive. Hier muss ich aber wohl mit USB verbunden sein.
![IMG20221129173124_copy_800x360](https://user-images.githubusercontent.com/43674377/204747473-86237f57-608d-4007-9cb1-de01162a2343.jpg)
Zudem kommt die Meldung, dass man eine Connected App benutzen muss. Es gibt ein solche aber nicht im PlayStore. Ggf. ist die MyBMW App der Nachfolger. Aber wenn ich die starte erscheint nichts weiters auf dem BMW Display.

Wo liegt das Problem, dass sich das Handy via MyBWM App und AAidrive App nicht per T mit dem Wagen verbindet

**Comment by 6i6i** [comment]
Hattest du mal versucht das Handy zusätzlich über USB zu verbinden? Die MyBMW App sollte ausreichend sein.

**Comment by vocaris9** [comment]
Ja habe ich. Wenn ich es anstecke kommt erst die Frage vom Smatphone ob ich nur laden oder auch Daten übertragen will. Ich bestätige Daten. Kurz später sehe ich im Connected Drive Menü das Smartphone als Gerät und der Hinweis, die BMW Connected App zu laden, die es nicht mehr gibt.
![IMG20221130114241](https://user-images.githubusercontent.com/43674377/204776795-8e5ab361-bb90-4c79-9a8d-7a2692af0874.jpg)
Auf dem Smartphon erscheint die Meldung, dass die my BMW App verbunden ist. In de rapp selber sehe ich nicht mehr als vorher. Auch beim km Stand steht --.
Wechsel ich auf Verbindungen, sehe ich das USB Gerät zu den BT Geräten
![IMG20221130114213](https://user-images.githubusercontent.com/43674377/204777447-14dced5f-15d4-47bf-8ac8-c9412078a351.jpg)
Wähle ich es aus kommt der Hinweis den USB Modus zu aktivieren. Der ist aber an.
![IMG20221130114224](https://user-images.githubusercontent.com/43674377/204777513-599bf9dc-40e1-40e2-9842-b72788b3fc9b.jpg)
Sonst passiert nichts.
Installiere ich jetzt die AAidrive App und lasse die USB Verbindung bestehen, kommt nach einiger Zeit kurze eine Verbindung (zu sehen auf der App) zu stande. Die hält 5 Sekunden und ist wieder weg. Und das geht dann so weiter. aber auf dem Display im Auto ist nichts zu sehen. Also keine Änderung

**Comment by 6i6i** [comment]
Die Einstellung "nur Laden" sollte ok sein. 
" Kurz später sehe ich im Connected Drive Menü das Smartphone als Gerät und der Hinweis, die BMW Connected App zu laden, die es nicht mehr gibt." das ist normal und wird bei mir auch angezeigt. Es funktioniert danach aber trotzdem.
Evtl mal am Smartphone versuchen die Akkuoptimierung und Arbeitsspeicheroptimierung von MyBMW und AAIdrive zu deaktiveren.

**Comment by hufman** [comment]
MyBMW is buggy about USB connections, and I have seen that repeated disconnect and reconnect problem.
Feel free to try installing BMW Connected (don't even need to be signed in) or BMW Connected Classic to use the USB functionality. Does MyBMW's remote functionality still work for your car? If not, you don't need it to be installed while using the older BMW Connected.

**Comment by vocaris9** [comment]
But there is no app called BMW Connection in the Play Store.

**Comment by hufman** [comment]
Yes indeed, you'll have to look on an APK mirror site, like https://apkpure.com/search?q=bmw+connected

**Comment by vocaris9** [comment]
Ok. Must i delead thr mybmw app before?
And is this the solution only fur USB connect?
Is it a Problem, that my F36 does not have 6NG?

**Comment by hufman** [comment]
You should delete MyBMW to reduce confounding variables during testing. Does MyBMW provide any other functionality for your car?
BMW Connected merely provides a more reliable USB connection. MyBMW has the same Bluetooth reliability as BMW Connected.
Your car's BMW Connected screens look correct, and since AAIdrive can sometimes connect, all the necessary vehicle options are in place.

**Comment by 6i6i** [comment]
But be carefull that you not take the newest version from BMWConnected:

_"I can also confirm, that BMWConnected Application Veresion 6.5.0.6411 breaks the complete Phone <-> Car Connection. The phone is shown as "not supported" in Idrive.
I tried to use the new MyBMW App. This one crashes immediatelly after startup every time it is launched. I downgraded to BMWConnected 6.4.1.6349 using APKPure and the Connection is working again."_

**Comment by LACNR** [comment]
In your case the problem might been that your car is from 2014 which can be supported by MyBMW but must not be functional enough. Your most reliable choice would be BMW Connected Classic or BMW Connected all found on alternate app download (e.g. apkmirror) because they were replaced by MyBMW. 

Another difficulty is that your phone is asked to choose the connection mode instead of automatically selecting the ideal mode although when select correctly it should not interfer cause the BMW Connected App will solve the connection.

Eventually look for a Software Update for your car  https://www.bmw.com/en/footer/software-updates.html
